ci-operator/config/openshift/hypershift-oadp-plugin/openshift-hypershift-oadp-plugin-main.yaml

```diff
@@ -37,6 +37,14 @@ tests:
       SNYK_CODE_ADDITIONAL_ARGS: --severity-threshold=high -d
       SNYK_ENABLE_DEPS_SCAN: "false"
     workflow: openshift-ci-security
+- as: build
+  commands: make local
+  container:
+    from: src
```

Comment:
nit: This works for now, but if we ever need to use this binary build in a future test, we should create a `Dockerfille` and build a proper image for the pipeline.

Reply:
Agree, this is just to validate it builds properly. I think we will need to integrate with Konflux dor build + publishing. Is that right?